docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+<!-- Thank you for submitting this PR. You are awesome!
+-->
+
+**Status**:
+- [ ] Ready
+- [ ] In Development
+- [ ] On Hold
+
+**Which issue does this PR fix?**: fixes #
```

Comment:
Add this line in the description "Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber"